lib/kitchen/directions/bake_chapter_content_module.rb

```diff
@@ -0,0 +1,13 @@
+module Kitchen
+  module Directions
+    module BakeChapterContentModule
+      def self.v1(book:)
+        book.chapters.each do |chapter|
+          chapter.search('[data-type="abstract"]').each do |abstract|
```

Comment:
wherever you end up putting the search, on chapter, page, or book, make an `abstract` or `abstracts` method to encapsulate this CSS selector, e.g. https://github.com/openstax/kitchen/blob/master/lib/kitchen/chapter_element.rb#L42-L44.  Also make a spec for that method.

Reply:
This may be moot depending on how you implement other parts of this review, but in this case where `chapter` is only used once, you don't need to get each chapter and then search on it -- you can just search on `.chapters`, like

```ruby
book.chapters.search(...).each do |abstract|
```